lib/src/main/java/com/thenewboston/api/common/PostDataSource.kt

```diff
@@ -94,6 +94,15 @@ class PostDataSource @Inject constructor(private val networkClient: NetworkClien
         return Outcome.Success("Successfully sent connection requests")
     }
 
+    suspend fun doSendPrimaryValidatorUpdated(request: ConnectionRequest): Outcome<String> {
+        networkClient.defaultClient.post<HttpResponse> {
+            url(ConfirmationValidatorAPIEndpoints.PRIMARY_VALIDATOR_UPDATED)
+            body = request
+        }
+
+        return Outcome.Success("Successfully sent primary validator updated")
```

Comment:
I was thinking we could give a better message like "Successfully updated primary validator"